### ISSUE TITLE
Issue running sm64coopdx_v1.0.1_macOS_Intel.dmg.zip - Fatal Error could not init GLEW Missing GL Version

### ISSUE THREAD

**Issue by futurepr0n** [coop-deluxe/sm64coopdx#229]
**Describe the bug**
I am getting an error when starting the app up. It fails to initialize GLEW.

**To Reproduce**
Steps to reproduce the behavior: (after installing the dmg)
1. Go to 'Applications'
2. Click on 'sm64coopdx'

**Expected behavior**

**Screenshots**
![Screenshot 2024-07-19 at 7 12 17 PM](https://github.com/user-attachments/assets/109b5677-0025-4539-a330-f59e2b5ca9d7)

**Desktop (please complete the following information):**

 - OS:: Mac
 - Browser Safari
 - Version Sonoma 14

**Additional context**
Basically just running the release, after installing the dmg, and it comes up with the fatal error on the window. Any step missing?

**Comment by EmeraldLoc** [comment]
Unable to reproduce, don't really know what the problem with this would be, since libs are added into the app file, and you're running macOS 14. Glew hasn't been updated in a very long time, so I don't think it's the lib. Will need to get a VM to test this out. In the meantime, you can compile the repo manually.

**Comment by futurepr0n** [comment]
heres the thing, the dependencies for audiofile on homebrew failed:

`Error: audiofile has been disabled because it is not maintained upstream!`

Is there a way to get it?

**Comment by EmeraldLoc** [comment]
Gotta love this problem. Do what [this guy says](https://stackoverflow.com/questions/73586208/can-you-install-disabled-homebrew-packages), except do it for libaudiofile

**Comment by futurepr0n** [comment]
Ok, thanks, compiling prior brought 

![Screenshot 2024-07-19 at 7 58 15 PM](https://github.com/user-attachments/assets/aeb1b978-6c20-47b7-ae56-685bf0732b7d)
I wasnt exactly able to edit the existing audiofile, because i didnt have it previously installed, so it failed to edit at first but looked like it changed me to be in developer mode. I tried the command again but nothing happened, so then I attempted running:

`HOMEBREW_NO_INSTALL_FROM_API=1 brew install audiofile`

This threw the same error, but now when I ran the edit command I was able to get the screen needed to edit the line successfully.

I then ran the gmake clean and the gmake command again to build, but I got the same error:
![Screenshot 2024-07-19 at 8 29 19 PM](https://github.com/user-attachments/assets/b5852b13-1878-4e3f-b4c2-ca5b70a62f97)

**Comment by futurepr0n** [comment]
lol, any chance you got the next error before I run into it? jk - thank you - am retrying will update this comment and close this item after success - appreciate the help

**Comment by futurepr0n** [comment]
After compiling, going to build and opening the package, same error thrown, wierd - 
![Screenshot 2024-07-19 at 11 06 30 PM](https://github.com/user-attachments/assets/06dc3bb4-8537-41b9-83a1-e02113c8f10d)
Does this require any special permissions? or am I doing something wrong how else other than just double clicking it am I supposed to run this?

**Comment by EmeraldLoc** [comment]
Hmmm, I don't know. Very interesting you got this error on a fresh compilation. No special perms are required, so don't know why this is happening, very odd...

**Comment by futurepr0n** [comment]
dude, i feel so dumb, I just remembered - I have the gfxCardStatus installed, so I can switch between discrete and integrated gpu and if its set to integrated gpu it will not work. It runs fine both the release and the build - I just need to make sure im flipped to discrete GPU. Jeez,, macbookpros - its set to autoswitch, but it doesnt kick in for some reason when this program tries to run. Sorry about this.
![Screenshot 2024-07-19 at 11 10 30 PM](https://github.com/user-attachments/assets/e1cfe649-50a8-4388-81be-0814f165db7a)

**Comment by EmeraldLoc** [comment]
Lol, nah all good.
